docs/reference/module-overview.md

```diff
@@ -73,13 +71,17 @@ modules.export = {
 };
 ```
 
-You should not use both `improve` and `extend` in a single module.
+You should not use both `improve` and `extend` in a single module. If "improving" an existing module, that existing module has already taken care of the "extending."
```

Comment:
Sure, but it's still "extending." I'm (I think) intentionally not explicitly saying that it will use `extend`, but it's still doing the thing implicitly.

Reply:
I'll not in `extend` that leaving it off implicitly extends the base module.